doc/src/suite-config.rst

```diff
@@ -1068,31 +1063,7 @@ where suite initial and final cycle points can be assumed. Some examples:
                      # (short for R1/P0/final-point)
 
 
-Example
-'''''''
-
-The tutorial illustrates integer cycling in :ref:`TutInteger`, and
-``<cylc-dir>/etc/examples/satellite/`` is a
-self-contained example of a realistic use for integer cycling. It simulates
-the processing of incoming satellite data: each new dataset arrives after a
-random (as far as the suite is concerned) interval, and is labeled by an
-arbitrary (as far as the suite is concerned) ID in the filename. A task called
-``get_data`` at the top of the repeating workflow waits on the next
-dataset and, when it finds one, moves it to a cycle-point-specific shared
-workspace for processing by the downstream tasks. When ``get_data.1``
-finishes, ``get_data.2`` triggers and begins waiting for the next
-dataset at the same time as the downstream tasks in cycle point 1 are
-processing the first one, and so on. In this way multiple datasets can be
-processed at once if they happen to come in quickly. A single shutdown task
-runs at the end of the final cycle to collate results. The suite graph is
-shown in :numref:`fig-satellite`.
-
-.. _fig-satellite:
-
-.. figure:: graphics/png/orig/satellite.png
-   :align: center
-
-   The ``etc/examples/satellite`` integer suite.
+.. TODO: INTEGER CYCLING EXAMPLE REMOVED - REPLACE IT?  
```

Comment:
Yeah, an example here wouldn't hurt.

Do you want to punt this down the line for now?

Reply:
I've added the pipeline example from the CiSE paper.